rest-api/spring-rest/src/main/java/uk/gov/gchq/gaffer/rest/controller/GraphConfigurationController.java

```diff
@@ -96,7 +96,7 @@ public Set<Class> getFilterFunctions(@PathVariable("inputClass") final String in
             try {
                 predicate = (Predicate) predicateClass.newInstance();
             } catch (final IllegalAccessException | InstantiationException e) {
-                LOGGER.warn("Failed to create new instance of " + predicateClass, e);
+                LOGGER.warn("Failed to create new instance of {}", predicateClass, e);
```

Comment:
Like above, this is using `e` as a `Throwable` so this won't work. A good tell is that there are more arguments than `{}`s. You can use the same solution as above with `String.format`